apps/avifenc.c

```diff
@@ -562,6 +575,24 @@ int main(int argc, char * argv[])
         } else if (!strcmp(arg, "-k") || !strcmp(arg, "--keyframe")) {
             NEXTARG();
             keyframeInterval = atoi(arg);
+        } else if (!strcmp(arg, "--q")) {
+            NEXTARG();
+            quality = atoi(arg);
+            if (quality < AVIF_QUALITY_WORST) {
```

Comment:
maybe use AVIF_CLAMP for shorter code.

Reply:
AVIF_CLAMP, AVIF_MIN, and AVIF_MAX are internal macros. This is why we can't use them in the apps/ directory, unfortunately.

We can certainly define equivalent macros in some header in apps/shared/ and use them here.